.github/workflows/update-documentation.yaml

```diff
@@ -19,9 +19,8 @@ jobs:
             matrix:
                 version:
                     - { ref: '2.x', version: 'latest' }
+                    - { ref: '2.3', version: '2.3' }
                     - { ref: '2.2', version: '2.2' }
-                    - { ref: '2.1', version: '2.1' }
-                    - { ref: '2.0.12', version: '2.0' }
```

Comment:
This ensures, that no new builds are triggered for the `2.1` and `2.0` versions, but they still remain in the version picker, because of the `AVAILABLE_VERSIONS` env variabl above